src/specBuilder/bar/barUtils.ts

```diff
@@ -254,6 +256,44 @@ export const getStroke = ({ name, children, color, colorScheme }: BarSpecProps):
 	];
 };
 
+export const getDimensionSelectionRing = (props: BarSpecProps): RectMark => {
+	const { name, colorScheme, paddingRatio, orientation } = props;
+
+	const update =
+		orientation === 'vertical'
+			? {
+					y: { value: -5 },
+					y2: { signal: 'height + 5' },
+					xc: { signal: `scale('xBand', datum.${name}_selectedGroupId) + bandwidth('xBand')/2` },
+					width: { signal: `bandwidth('xBand')/(1 - ${paddingRatio} / 2)` },
+			  }
+			: {
+					x: { value: -5 },
+					x2: { signal: 'width + 5' },
+					yc: { signal: `scale('yBand', datum.${name}_selectedGroupId) + bandwidth('yBand')/2` },
+					height: { signal: `bandwidth('yBand')/(1 - ${paddingRatio} / 2)` },
+			  };
+
+	return {
+		name: `${name}_selectionRing`,
+		type: 'rect',
+		from: {
+			data: `${name}_selectedData`,
+		},
+		interactive: false,
+		encode: {
+			enter: {
+				fill: { value: 'transparent' },
+				strokeWidth: { value: 2 },
+				stroke: { value: getColorValue('static-blue', colorScheme) },
+				cornerRadius: { value: 6 },
+				padding: { value: 5 },
```

Comment:
Does this do anything? Looking at the mark docs for `rect` I don't see anything about padding.
https://vega.github.io/vega/docs/marks/rect/